manifests/init.pp

```diff
@@ -69,7 +69,7 @@
   }
 
   if $iptables_docker_chain {
-    include 'iptables'
+    include '::iptables'
```

Comment:
I thought the Puppet best practices/style guide did not want :: in front of class?  Why add it here?

Reply:
My linter has been yelling at me to do it so I have been